src/dimension_slice.c

```diff
@@ -759,10 +760,11 @@ dimension_slice_check_chunk_stats_tuple_found(TupleInfo *ti, void *data)
 }
 
 int
-ts_dimension_slice_oldest_chunk_without_executed_job(int32 job_id, int32 dimension_id,
-													 StrategyNumber start_strategy,
-													 int64 start_value, StrategyNumber end_strategy,
-													 int64 end_value)
+ts_dimension_slice_oldest_valid_chunk_for_job_execution(int32 job_id, int32 dimension_id,
```

Comment:
This is the reorder specific function. Lets rename this.
```suggestion
ts_dimension_slice_oldest_valid_chunk_for_reorder(int32 job_id, int32 dimension_id,
```